testutil-server/src/main/java/io/spine/testing/server/blackbox/VerifyMessages.java

```diff
@@ -26,7 +26,7 @@
  * @param <E> the type of emitted messages
  * @author Alexander Yevsyukov
  */
-public interface Verify<E extends EmittedMessages> {
+public interface VerifyMessages<E extends EmittedMessages> {
```

Comment:
Not sure why this got renamed. It was named `Verify` to make the code usages to read fluently, i.e. "to verify<what>".